cl/search/fixtures/florida_state_court_data.json

```diff
@@ -0,0 +1,3726 @@
+[
+  {
+    "fields": {
+      "jurisdiction": "ST",
+      "position": 355.05,
+      "short_name": "",
+      "full_name": "Florida Judge's Courts",
+      "citation_string": "",
```

Comment:
I think `citation_string` is used in a few places and we'll need to include it. Can you check?

Reply:
Added citation strings for all.